app-ios/Sources/Strings/Resources/en.lproj/Localizable.strings

```diff
@@ -3,7 +3,7 @@
 // About
 "About.Title" = "DroidKaigiとは";
 "About.WhatIsDroidKaigi" = "What is DroidKaigi?";
-"About.Description" = "DroidKaigiはエンジニアが主役のAndroidカンファレンスです。\nAndroid技術情報の共有とコミュニケーションを目的に、2022年10月5日(水)〜7日(金)の3日間開催します。";
+"About.Description" = "DroidKaigiはエンジニアが主役のカンファレンスです。\n技術情報の共有とコミュニケーションを目的に、2022年10月5日(水)〜7日(金)の3日間開催します。";
```

Comment:
Now that we merge https://github.com/DroidKaigi/conference-app-2022/pull/387, I think we can use them. But it is OK now. 🙏

Reply:
I'll check that after this PR merged 🙏🏼